src/api/app/controllers/webui/packages/bs_requests_controller.rb

```diff
@@ -13,7 +13,11 @@ def index
         session[:request_numbers] = requests_query.requests.map(&:number)
 
         respond_to do |format|
-          format.json
+          if switch_to_webui2?
+            format.json { render 'webui2/shared/bs_requests/index' }
+          else
+            format.json
+          end
```

Comment:
Interesting way of doing this :)

Reply:
The `switch_to_webui`  doesn't seem to work when using a respond_to block:/